docker-compose.yml

```diff
@@ -0,0 +1,204 @@
+# This is a Docker Compose development environment for Restarters.  It:
+# - relies on you setting up www.example.com in your hosts file.  TODO this name could be better but needs some configuration.
+# - a web server on http://www.example.com:8001 which serves up restarters.  Admin user is jane@bloggs.net/passw0rd.
+# - phpMyAdmin on http://www.example.com:8002.  Use host restarters_db, user root/s3cr3t
+# - Discourse on http://www.example.com:8003.  User is someuser/mustbetencharacters; see below about hostname.
+#   TODO This is stock Discourse - no theme.  Would be nice to have the theme, but that doesn't matter so far as
+#   testing the integration.
+# - Mailhog on http://localhost:8025.
+#
+# TODO If your host is Linux, this probably only works if you run as root.
```

Comment:
I got confused a bit by this, suggest change to perhaps: 

>    On the machine where you will be using your browser to access the Restarters / Discourse sites once set up on Docker, add a record to your hosts file to point the domain restarters.docker to the machine where you are installing Docker.

(then use restarters.docker instead of www.example.com in the rest of the docs)

Reply:
I'll add the comment, but I want to stick with www.example.com for now as it's the default configuration in (I think) the Discourse container, and I don't want to need to do another tear-down and retest if possible.

Reply:
A-ha, yes no problem - didn't realise it was used internally.